src/azure-cli/azure/cli/command_modules/ams/_utils.py

```diff
@@ -60,22 +60,22 @@ class JsonBytearrayEncoder(json.JSONEncoder):
     DATE_FORMAT = "%Y-%m-%d"
     TIME_FORMAT = "%H:%M:%S"
 
-    def default(self, obj):  # pylint: disable=E0202,W0221
```

Comment:
`W0221` is `arguments-differ`: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/arguments-differ.html